lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -83,9 +84,10 @@ class StandardAccountTest {
     }
 
     @Test
-    fun `creating account with private key`() {
-        val privateKey = Felt(1234)
-        StandardAccount(Felt.ZERO, privateKey, provider, chainId)
+    fun creatingAccountWithPrivateKey() {
+        val address = Felt(123)
+        val privateKey = Felt(456)
+        val account = StandardAccount(Felt.ZERO, privateKey, provider, chainId)
```

Comment:
Address is not used. I assume you wanted to use it like so to make example more self-explanatory?
```suggestion
        val address = Felt(123)
        val privateKey = Felt(456)
        val account = StandardAccount(address, privateKey, provider, chainId)
```